test/common/log.test.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as assert from "assert";
+import * as path from "path";
+import * as fs from "fs";
+import { getLoggingDirectory } from "../../src/extension/log/LogHelper";
+
+suite("logHelper", function() {
+    suite("commonContext", function() {
+        const REACT_NATIVE_TOOLS_LOGS_DIR = process.env.REACT_NATIVE_TOOLS_LOGS_DIR;
```

Comment:
There is no reason for storing previous values of process.env here - they live only in process context and when you redefine them it affects only process.env not the global environment.

Reply:
I understand that, but maybe we will need this values somewhere else, so I prefer to avoid side effects